### PR TITLE
Final logging and error handling fixes

### PR DESCRIPTION
**Pull request by tjclavel**
## Description of the change

> Description here

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Cleanup
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Test Plan

- [ ] Unit test
- [ ] Tested locally

### CODE REVIEW

File: src/IFrameElements/constants.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-export const ELEMENTS_PREVIEW_DEV_URL = 'https://capital.widgets.dev.parafin.com'
-export const ELEMENTS_PREVIEW_URL = 'https://capital.widgets.parafin.com'
+export const ELEMENTS_PREVIEW_DEV_URL = 'https://www.capital.widgets.dev.parafin.com'
```
Comment (reedhodder) [inline src/IFrameElements/constants.ts:1]: why is `www.` necessary?

File: src/IFrameElements/helpers/utils.ts [hunk 2]
```diff
@@ -58,15 +58,20 @@ export const buildDashboardIFrame = (id: string, url: string, token: string) =>
...
   try {
+    // TODO: Let's get rid of this eventually
     const userId = JSON.parse(atob(token.split('.')[1]))[environment === 'production'
       ? 'https://api.parafin.com/user_authorization'
       : 'https://api.dev.parafin.com/user_authorization'
     ].app_metadata.userId
     return userId
   } catch (err: any) {
-    // TODO: Report w/ sentry
+    report(`Unexpected error: Unable to pull userId out of JWT (${token})`)
```
Comment (reedhodder) [inline src/IFrameElements/helpers/utils.ts:74]: this message will be massive with the bearer token in it, can we do metadata for it instead?

Comment (reedhodder) [reply src/IFrameElements/helpers/utils.ts:74]: I think the errorHandler has a `setContext` or something like that

File: src/IFrameElements/constants.ts (hunk 1 above)
Comment (tjclavel) [reply src/IFrameElements/constants.ts:1]: TBH Idk I was just having weird redirect issues from hosting on amplify

Comment (tjclavel) [reply src/IFrameElements/constants.ts:1]: I can make a todo to try to figure this out later but for now I wanna just not get blocked on weird hosting stuff

File: src/IFrameElements/helpers/utils.ts (hunk 2 above)
Comment (tjclavel) [reply src/IFrameElements/helpers/utils.ts:74]: good point

File: src/IFrameElements/helpers/utils.ts [hunk 3]
```diff
@@ -58,15 +58,22 @@ export const buildDashboardIFrame = (id: string, url: string, token: string) =>
...
+  token: string,
+  environment: Environment,
+  capture: (message: string, contexts?: { contexts: any }) => void
+) => {
   try {
+    // TODO: Let's get rid of this eventually
     const userId = JSON.parse(atob(token.split('.')[1]))[environment === 'production'
       ? 'https://api.parafin.com/user_authorization'
-      : 'https://api.dev.parafin.com/user_authorization'
+      : 'https://api.dev.parafin.com/user_authorizatio'
```
Comment (reedhodder) [inline src/IFrameElements/helpers/utils.ts:70]: typo

**Review by reedhodder — APPROVED** [review]

File: src/IFrameElements/helpers/utils.ts (hunk 3 above)
Comment (tjclavel) [reply src/IFrameElements/helpers/utils.ts:70]: hahahah thank you this is how I was testing broken JWT format 🤦‍♂️

**Pull request merged**